PortSpeedTestTemplate/NetworkCardSpeedTest.cs

```diff
@@ -0,0 +1,114 @@
+﻿//Company's property 2024
+
+namespace PortSpeedTestTemplate
+{
+    /// <summary>
+    /// Network card speed test 
+    /// </summary> 
+    public class NetworkCardSpeedTest : NetworkCardSpeedTestFlow
+    {
+        #region Fields
+
+        /// <summary>
+        /// Name of test
+        /// </summary>
+        public const string Name = "WebCardSpeedTest";
+
+        /// <summary>
+        /// Description of test
+        /// </summary>
+        public const string Description = "Test with mode";
+
+        #endregion
+
+        #region Data set
```

Comment:
The region is about type of members inside, and not about it's name. It should be Properties